packages/shared/src/components/auth/AccountCompletionModals.tsx

```diff
@@ -1,105 +0,0 @@
-import { useRouter } from 'next/router';
-import React, { ReactElement, useContext } from 'react';
-import { useQuery } from 'react-query';
-import AuthContext from '../../contexts/AuthContext';
-import useProfileForm from '../../hooks/useProfileForm';
-import { useToastNotification } from '../../hooks/useToastNotification';
-import useWindowEvents from '../../hooks/useWindowEvents';
-import {
-  AuthEvent,
-  AuthFlow,
-  ErrorData,
-  ErrorEvent,
-  getKratosError,
-  getKratosFlow,
-  KRATOS_ERROR,
-} from '../../lib/kratos';
-import { StyledModal } from '../modals/StyledModal';
-import IncompleteRegistrationModal from './IncompleteRegistrationModal';
-import {
-  SocialRegistrationForm,
-  SocialRegistrationFormValues,
-} from './SocialRegistrationForm';
-
-function AccountCompletionModals(): ReactElement {
-  const router = useRouter();
-  const { displayToast } = useToastNotification();
-  const { user, shouldShowLogin, logout, refetchBoot } =
-    useContext(AuthContext);
-  const {
-    updateUserProfile,
-    hint,
-    onUpdateHint,
-    isLoading: isProfileUpdateLoading,
-  } = useProfileForm({
-    onSuccess: () => refetchBoot(),
-  });
-  const onUpdateProfile = ({ file, ...params }: SocialRegistrationFormValues) =>
-    updateUserProfile(params);
-
-  const displayErrorMessage = (text: string) => {
-    displayToast(text);
-    window.history.replaceState(null, '', '/');
-  };
-
-  useQuery(
-    [{ type: 'login', flow: router?.query.flow as string }],
-    ({ queryKey: [{ flow }] }) => getKratosFlow(AuthFlow.Recovery, flow),
-    {
-      enabled: !!router?.query.recovery && !!router?.query.flow,
-      onSuccess: (data) => {
-        if ('error' in data) {
-          const errorData = data as unknown as ErrorData;
-          displayErrorMessage(errorData.error.message);
-          return;
-        }
-
-        if (!data?.ui?.messages?.length) {
-          return;
-        }
-
-        const error =
-          data.ui.messages.find(
-            (message) => message.id === KRATOS_ERROR.INVALID_TOKEN,
-          ) || data.ui.messages[0];
-        displayErrorMessage(error.text);
-      },
-    },
-  );
```

Comment:
This one is used to handle error messages from expired recovery flow when they get redirected to the app.